VirtualList/useThemeVirtualList.module.less

```diff
@@ -0,0 +1,14 @@
+// useThemeVirtualList.module.less
+//
+@import '../styles/skin.less';
+@import '../styles/variables.less';
+
+.scaled {
+	transform: @sand-imageitem-vertical-focus-image-transform;
+}
+
+.applySkins({
+	.scaled {
+		box-shadow: @sand-imageitem-focus-bg-shadow;
+	}
+});
```

Comment:
Why don't we remove this styles on ImageItem.module.less?

Reply:
👍